react/features/large-video/components/LargeVideo.native.js

```diff
@@ -107,16 +112,19 @@ class LargeVideo extends Component<Props, State> {
             avatarSize,
             useConnectivityInfoLabel
         } = this.state;
+        const { onPress } = this.props;
 
         return (
             <DimensionsDetector
                 onDimensionsChanged = { this._onDimensionsChanged } >
                 <ParticipantView
                     avatarSize = { avatarSize }
+                    onPress = { onPress }
```

Comment:
In this file `onPress` is decomposed out of `this.props` and use on this line only. On the next line we have `this.props._participantId`. Let's have `this.props.onPress` then.

Reply:
Right, I didn't see that sorry. However if we decompose nearly everything above 1 variable (see this.state a couple of lines above) then I guess it's more consistent to decompose props too so then I'd rather change _participantId than onPress. What do you think?

Reply:
Whatever, just be consistent.